components/UserCard.tsx

```diff
@@ -23,9 +28,17 @@ iframe {
 }
 `;
 
-export const UserCard = () => {
-  const { signer, user, account } = useSignerContext();
-  const { provider } = useAccountContext();
+export const UserCard = ({
+  account,
+}: {
+  account: MultiOwnerModularAccount<AlchemySigner>;
+}) => {
+  const bundlerClient = useBundlerClient();
+  const signer = useSigner();
+  const provider = useSmartAccountClient({
+    account,
+  });
```

Comment:
I'm wondering if I should make this `async` so that it can take in the result of `useAccount` when it's in the same component. as it stands, `useSmartAccountClient` has to be used downstream of `useAccount` because you need to await the initialization of the account